docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -1171,6 +1170,39 @@ Retrieve the value of the key `foo` after recreating the service state from the
 dagger call get --key=foo
 ```
 
+### Execute functions concurrently
+
+The following Dagger Function demonstrates how to use native-language concurrency features ([goroutines](https://gobyexample.com/goroutines) in Go, [task groups](https://docs.python.org/3/library/asyncio-task.html) in Python), and [promises](https://basarat.gitbook.io/typescript/future-javascript/promise) in TypeScript to execute other Dagger Functions in parallel.
```

Comment:
The AnyIO TaskGroup is [not the same](https://anyio.readthedocs.io/en/stable/tasks.html#differences-with-asyncio-taskgroup) as the AnyIO TaskGroup: https://anyio.readthedocs.io/en/stable/tasks.html

Also, parallelization is not the same as concurrency:

```diff
- to execute other Dagger Functions in parallel.
+ to execute other Dagger Functions concurrently.
```

Reply:
Done